src/lib/components/chatInput/ModelPickerPopover/ModelPickerPopoverContent.svelte

```
@@ -0,0 +1,127 @@
+              selectedModel = model;
+              onSelect?.(model);
+              open = false;
+            }}
+          >
+            <ModelPickerModelEntry {model} />
+          </Command.Item>
+        {/each}
+      </Command.Group>
+    {:else}
+      {@const viewModels = MODELS.filter((model) => !MODEL_DETAILS[model].free)}
+      {@const freeModels = viewModels.filter((model) => FREE_MODELS.includes(model as any))}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)